### PR TITLE
Replace Synology DSM services with buttons

### PR DESCRIPTION
**Pull request by mib1185**
## Proposed change

The services `reboot` and `shutdown` are deprecated and will be removed in future release. Please use the new button entities.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core/pull/57352
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by frenck** [comment]
not stale bot... thanks :)

**Comment by mib1185** [comment]
TBH i do not have any clue about this CI jobs 🙈 

This is the only error I see in the respective log

https://app.netlify.com/sites/home-assistant-docs/deploys/61ef1a5b5f85290008285977
```
[...]
10:35:44 PM: JSON::ParserError: 783: unexpected token at '<html><body>You are being <a href="https://community.home-assistant.io/l/top?period=all">redirected</a>.</body></html>'
10:35:44 PM: /opt/build/repo/Rakefile:225:in `block in <top (required)>'
/opt/build/cache/bundle/ruby/2.7.0/gems/rake-13.0.6/exe/rake:27:in `<top (required)>'
/opt/build/cache/bundle/ruby/2.7.0/bin/ruby_executable_hooks:24:in `eval'
/opt/build/cache/bundle/ruby/2.7.0/bin/ruby_executable_hooks:24:in `<main>'
Tasks: TOP => blueprint_exchange_data
[...]
```

**Comment by frenck** [comment]
Rebased the PR to deal with the CI issues

File: source/_integrations/synology_dsm.markdown [hunk 1]
```diff
@@ -103,20 +103,37 @@ A switch is available to enable/disable the [Surveillance Station](https://www.s
...
+
 ## Services
 
+<div class='note warning'>
+
+The services `synology_dsm.reboot` and `synology_dsm.shutdown` are deprecated and will be removed in future release.
+Please use the new button entities.
+
+</div>
+
```
Comment (frenck) [inline source/_integrations/synology_dsm.markdown]: We don't keep a log of deprecations in our documentation.
Please remove this note and the deprecated services from the documentation instead.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/synology_dsm.markdown (hunk 1 above)
Comment (mib1185) [reply source/_integrations/synology_dsm.markdown]: yeah, makes absolutely sense 👍

**Review by frenck — APPROVED** [review]
Thanks, @mib1185 👍

**Pull request merged**